corehq/apps/app_manager/models.py

```diff
@@ -5811,13 +5812,15 @@ def import_app(app_id_or_source, domain, source_properties=None):
     if isinstance(app_id_or_source, six.string_types):
         app_id = app_id_or_source
         source = get_app(None, app_id)
-        src_dom = source['domain']
+        source_domain = source['domain']
         source = source.export_json(dump_json=False)
+        report_map = get_static_report_mapping(source_domain, domain)
     else:
         cls = get_correct_app_class(app_id_or_source)
         # Don't modify original app source
         app = cls.wrap(deepcopy(app_id_or_source))
         source = app.export_json(dump_json=False)
+        report_map = {}
```

Comment:
Why is this blank?

(Not blocking, since this is the code block is used for the import page not the copy app workflow.)

Reply:
I'm also curious about this - `import_app` accepts `app_id_or_source`, and this means that the static UCRs are transferred if you pass in the app ID, but not if you pass in the source.  This seems like a non-sequitor.  Can I move the `report_map` instantiation outside of this if/else block?

Reply:
honestly i'm not sure. I wonder if the source domain gets scrubbed in some instance which is what caused me to add this. Feel free to change. sorry for not documenting :(

Reply:
That could be - looks like `export_json` removes `domain` under the hood.  I'll make some clarifications to the code.